src/nyc_trees/apps/home/templates/home/about_faq.html

```diff
@@ -12,93 +12,103 @@
 
 <h4>What is TreesCount! 2015?</h4>
 <p>
-    TreesCount! 2015 is NYC Parks&rsquo; effort to map and catalogue every street
-    tree on every block in New York City.
-    <a href="{% url 'registration_register' %}">Register</a>
-    to become a volun<b>treer</b> today!</p>
+    TreesCount! 2015 is NYC Parks&rsquo; initiative to map and assess every street tree on every block in New York City.
+    <a href="{% url 'registration_register' %}">Register</a> to become a volun<b>treer</b> today!
+</p>
 
 <h4>Do I need to know a lot about trees to get involved?</h4>
 <p>
-    No prior experience is required! We will help you learn everything you
-    need to know to map street trees.
+    No prior experience is required! We will help you learn everything you need to know to participate.
+</p>
+
+<h4>How do I become a voluntreer?</h4>
+<p>
+    Getting involved is easy! To get started, <a href="{% url 'registration_register' %}">register</a> on our
+    website and take our brief online training course. Then, RSVP to and attend a training event with NYC Parks
+    or one of our Census partner groups to gain hands-on knowledge about how to assess and map street trees.
 </p>
 
-<h4>How do I volunteer?</h4>
+<h4>What happens at a TreesCount! 2015 event?</h4>
+<p>
+    There are two kinds of events: training and mapping.
+</p>
 <p>
-    Getting involved is easy! To get started, register online and take our brief
-    virtual training course. Then RSVP and attend a field training event
-    with NYC Parks or with a Census partner group to gain hands-on field
-    knowledge on how to identify and map street trees.
+    Training events are designed to help you apply what you learned in the online training course.
+    The first hour of a training event is spent mastering the TreeKIT mapping method and learning to use the Treecorder software.
+    After the first hour, you will collect data while mapping in teams.
 </p>
 <p>
-    At the training event, we&rsquo;ll have
-    staff teaching you how to apply what you&rsquo;ve learned, answer any
-    questions you have, supply necessary mapping tools and a Tree Species
-    ID guide, and get you mapping right away with other newly trained
-    volunteers!
-    Start your online training <a href="{% url 'training_summary_pure' %}">here</a>!</p>
+    Mapping events are intended for volun<b>treers</b> who have already been to a training event.
+    No instruction period is included, but staff will be present to help you out if you get stuck.
+    When you arrive, you will be paired with another volunteer, check out equipment using your volun<b>treer</b> badge,
```

Comment:
That is not what was indicated in the text provided to us, so I am going to leave this as-is.